<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClient</class>
 <widget class="QMainWindow" name="MainWindowClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;仿宋&quot;;
font: 75 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>10</number>
        </property>
        <property name="text">
         <string>社区户籍人员管理</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="text">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>密  码</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_login">
        <property name="text">
         <string>登陆</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>IP</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_client">
        <property name="text">
         <string>启动客户端</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_test">
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_clientStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>未链接服务器</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../res/登陆界面.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_changePassword"/>
    <addaction name="action_importData"/>
    <addaction name="action_addData"/>
    <addaction name="action_alterData"/>
    <addaction name="action_exportData"/>
    <addaction name="action_saveData"/>
    <addaction name="action_backupDatabase"/>
    <addaction name="action_client"/>
    <addaction name="action_logout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_changePassword">
   <property name="text">
    <string>密码修改</string>
   </property>
  </action>
  <action name="action_importData">
   <property name="text">
    <string>数据导入</string>
   </property>
  </action>
  <action name="action_addData">
   <property name="text">
    <string>数据添加</string>
   </property>
  </action>
  <action name="action_alterData">
   <property name="text">
    <string>数据修改</string>
   </property>
  </action>
  <action name="action_exportData">
   <property name="text">
    <string>报表生成</string>
   </property>
  </action>
  <action name="action_IP">
   <property name="text">
    <string>IP设置</string>
   </property>
  </action>
  <action name="action_logout">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_saveData">
   <property name="text">
    <string>保存数据</string>
   </property>
  </action>
  <action name="action_backupDatabase">
   <property name="text">
    <string>备份数据库</string>
   </property>
  </action>
  <action name="action_client">
   <property name="text">
    <string>启动服务器</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
